nautobot/core/testing/views.py

```diff
@@ -381,15 +381,15 @@ def test_create_object_with_constrained_permission(self):
         def test_slug_autocreation(self):
             """Test that slug is autocreated through ORM."""
             # This really should go on a models test page, but we don't have test structures for models.
-            if self.slug_source is not None:
+            if hasattr(self.model, "slug") and self.slug_source is not None:
```

Comment:
As above - the presence or absence of `slug_source` should control this test sufficiently - non-slug models shouldn't have a `slug_source` on their tests.